README.md

```diff
@@ -161,6 +161,8 @@ Key commands
 - <kbd>.</kbd> – Start/stop `z`-spinning (speed/direction can be controlled with <kbd>0</kbd> / <kbd>Enter</kbd>)
 - <kbd>←</kbd>, <kbd>→</kbd>, <kbd>↑</kbd>, <kbd>↓</kbd> – Manual rotation
 - <kbd>1</kbd>, <kbd>2</kbd>, <kbd>3</kbd>, <kbd>4</kbd>, <kbd>5</kbd>, <kbd>6</kbd>, <kbd>7</kbd>, <kbd>8</kbd>, <kbd>9</kbd> – Manual rotation along coordinate axes
+- <kbd>Alt</kbd> + <kbd>a</kbd> – Set axes number format
+- <kbd>Alt</kbd> + <kbd>c</kbd> – Set colorbar number format
```

Comment:
What does `Alt` mean on Mac? Is it `option` or `command` -- it will be good to clarify it here.

Reply:
For what's it worth, we do use <kbd>Alt</kbd> already in https://github.com/glvis/glvis/?tab=readme-ov-file#advanced

Reply:
I'm not able to test but I did see other places that used <kbd>Alt</kbd> like Tzanio mentioned. It'd probably be good to mention it somewhere at the top.